ui-admin/src/study/participants/survey/PrintFormModal.tsx

```diff
@@ -34,7 +34,8 @@ const PrintFormModal = ({ survey, answers, resumeData }: DownloadFormViewProps)
   return <Modal show={true} fullscreen={true}>
     <Modal.Body className="m-0 p-0">
       <div className="d-print-none d-flex justify-content-center py-2">
-        <Link className="btn btn-secondary" to="..">Done</Link>
+        {/* @ts-ignore Link to type also supports numbers for back operations */}
+        <Link className="btn btn-secondary" to={-1}>Done</Link>
```

Comment:
curious: is there a behavior difference b/w '..' and -1?

Reply:
`..` navigates up one level in the route, `-1` just goes to the previous page, wherever that may be. I had to change this because taskId is specified in the URL query params now, and `..` causes those params to be stripped out, whereas -1 preserves them. I could have also kept using `..` by grabbing the query params and passing them back in as a prop

Reply:
Ahh, very interesting. Thanks